mage/build.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2020 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mage
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/gravitational/trace"
+	"github.com/magefile/mage/mg"
+)
+
+type Build mg.Namespace
+
+func (Build) All() {
+	mg.SerialDeps(Build.Go)
+}
+
+func buildBoxName() string {
+	return fmt.Sprint("gravity-build:", buildVersion)
+}
+
+// Go builds go binaries using consistent build environment.
+func (Build) Go() (err error) {
+	mg.Deps(Build.BuildContainer, Build.Selinux)
+
+	m := root.Target("build:go")
+	defer func() { m.Complete(err) }()
+
+	packages := []string{"github.com/gravitational/gravity/tool/gravity", "github.com/gravitational/gravity/tool/tele"}
+	if enterprise != "" {
+		if err = hasE(); err != nil {
+			return trace.Wrap(err)
+		}
+
+		packages = []string{"github.com/gravitational/gravity/e/tool/gravity", "github.com/gravitational/gravity/e/tool/tele"}
```

Comment:
Move these paths (and the ones above) into constants? I see they're used in multiple places.